bin/commands/contributors.rb

```diff
@@ -32,4 +32,12 @@
       end
     end
   end
+
+  g.desc 'Create name,email CSV list of all DCO signers'
+  g.command 'dco-csv' do |c|
+    c.action do |_global_options, options, _args|
+      org = GitHub::Organization.new(options)
+      puts org.commits(options).unique_dco_signers_csv
+    end
+  end
```

Comment:
There's an issue for exporting CSV in a generic way, https://github.com/opensearch-project/project-tools/issues/18. 

How about we change this to be called "emails" and output something non-structured, then implement the CSV thing generically?

Reply:
I'm still looking at what it would take to wire in more generic csv support.